src/benchmarks/micro/libraries/System.Formats.Tar/Perf.TarFile.cs

```diff
@@ -50,21 +51,45 @@ public void CreateFromDirectory_Path()
             File.Delete(_outputTarFilePath);
         }
 
+        [Benchmark]
+        public async Task CreateFromDirectory_Path_Async()
+        {
+            await TarFile.CreateFromDirectoryAsync(sourceDirectoryName: _inputDirPath, destinationFileName: _outputTarFilePath, includeBaseDirectory: false);
+            File.Delete(_outputTarFilePath);
+        }
+
         [Benchmark]
         public void ExtractToDirectory_Path() => TarFile.ExtractToDirectory(sourceFileName: _inputTarFilePath, destinationDirectoryName: _outputDirPath, overwriteFiles: true);
 
+        [Benchmark]
+        public Task ExtractToDirectory_Path_Async() => TarFile.ExtractToDirectoryAsync(sourceFileName: _inputTarFilePath, destinationDirectoryName: _outputDirPath, overwriteFiles: true);
+
         [Benchmark]
         public void CreateFromDirectory_Stream()
         {
             using MemoryStream ms = new MemoryStream();
             TarFile.CreateFromDirectory(sourceDirectoryName: _inputDirPath, destination: ms, includeBaseDirectory: false);
         }
 
+        [Benchmark]
+        public async Task CreateFromDirectory_Stream_Async()
+        {
+            await using MemoryStream ms = new MemoryStream();
+            await TarFile.CreateFromDirectoryAsync(sourceDirectoryName: _inputDirPath, destination: ms, includeBaseDirectory: false);
+        }
+
         [Benchmark]
         public void ExtractToDirectory_Stream()
         {
             using FileStream fs = File.OpenRead(_inputTarFilePath);
             TarFile.ExtractToDirectory(source: fs, destinationDirectoryName: _outputDirPath, overwriteFiles: true);
         }
+
+        [Benchmark]
+        public async Task ExtractToDirectory_Stream_Async()
+        {
+            await using FileStream fs = File.OpenRead(_inputTarFilePath);
```

Comment:
The file should be opened for async IO

```suggestion
            await using FileStream fs = new FileStream(_inputTarFilePath, FileMode.Open, FileAccess.Read, FileShare.Read, 4096, useAsync: true);
```